### ISSUE TITLE
Get Metrics of YoloX on Test set and export model files into tensorflow

### ISSUE THREAD

**Issue by rahuja23** [Megvii-BaseDetection/YOLOX#1331]
Hello 
I have trained the model yolox_s on coco128 dataset but now I would like to print out the model metrics (specifically: precision, recall and f1 score) on a test set and also export the saved model file into tensorflow format. Can someone please guide me how this can be done?

**Comment by zogojogo** [comment]
up

**Comment by rahuja23** [comment]
what?
